modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -164,14 +173,41 @@ def add_missing_claimant_info_error(field, attribute_name:)
       errors.add attribute_name, "if any claimant info is present, claimant #{field} must also be present"
     end
 
+    def validate_hearing_type_selection
+      return if board_review_hearing_selected? && includes_hearing_type?
+
+      if hearing_type_missing?
+        errors.add :form_data, "if 'boardReviewOption' 'hearing' is selected, a 'hearingType' must be selected"
+      elsif unexpected_hearing_type_inclusion?
+        errors.add :form_data, "if 'boardReviewOption' 'direct_review' or 'evidence_submission' is selected, " \
```

Comment:
Full paths to `boardReviewOption` and `hearingType` here, too.

Reply:
Done! Much better thanks.